storage/connect/CMakeLists.txt

```diff
@@ -443,12 +442,20 @@ ENDIF()
 IF(CONNECT_WITH_JDBC AND JAVA_FOUND AND JNI_FOUND)
   # TODO: Find how to compile and install the java wrapper classes
   # Find required libraries and include directories
+  INCLUDE(UseJava)
   SET (JAVA_SOURCES JdbcInterface.java)
-  add_jar(JdbcInterface ${JAVA_SOURCES})
+  add_jar(JdbcInterface ${JAVA_SOURCES} OUTPUT_DIR ${CMAKE_CURRENT_BINARY_DIR}/connect_jars)
+  get_target_property(_jarFile JdbcInterface JAR_FILE)
   INSTALL(FILES
-    ${CMAKE_CURRENT_SOURCE_DIR}/mysql-test/connect/std_data/JavaWrappers.jar
-    ${CMAKE_CURRENT_BINARY_DIR}/JdbcInterface.jar
-    DESTINATION ${INSTALL_MYSQLSHAREDIR} COMPONENT connect-engine)
+    ${_jarFile} DESTINATION ${INSTALL_MYSQLSHAREDIR} COMPONENT connect-engine-jdbc)
+  IF (RPM)
+    SET(CPACK_COMPONENTS_ALL ${CPACK_COMPONENTS_ALL} connect-engine-jdbc PARENT_SCOPE)
+    SET(CPACK_RPM_connect-engine-jdbc_PACKAGE_REQUIRES "java-11-openjdk-devel" PARENT_SCOPE)
+    SET(CPACK_RPM_connect-engine-jdbc_PACKAGE_RECOMMENDS "connect-engine libmariadb-java" PARENT_SCOPE)
```

Comment:
Add:
* postgresql-jdbc
* mysql-connector-java
(unsure of cross RPM availability so weaker is probably better).

Please pick the right [strength and direction here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/packaging_and_distributing_software/new-features-in-rhel-8_packaging-and-distributing-software).

Reply:
Will add additional packages. Have analysed already weaker is `RECOMMENDS`, instead of `SUGGESTS`.